### PR TITLE
Update product scope and user stories

### LINKED ISSUE #19: Update product scope and user stories
Update target user, user stories and NFRs

### PR DESCRIPTION
**Pull request by limrik**

### CODE REVIEW

**Comment by cliftonchee** [comment]
LGTM! Aligns with what we have discussed earlier. Thanks @limrik!

**Review by cliftonchee — APPROVED** [review]
LGTM! Aligns with what we have discussed earlier. Thanks @limrik! I'm okay to merge it. Feel free to update and play around with issues/project board!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -274,27 +274,52 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `*`      | user    | update or create new records in bulk                                               | process a family more efficiently                                                |
+| `*`      | user    | set recurring tasks                                                                | I do not have to keep scheduling recurring appointments                          |
+| `*`      | user    | retrieve past records or revert changes easily                                     | revert my changes if I accidentally delete or wrongly edit a patient’s records   |
+| `*`      | user    | select what information is available when I view the list of patients              | cater the view to my needs                                                       |
+| `*`      | user    | add notes to a patient                                                             | include other additional information                                             |
+| `*`      | user    | easily generate reports of the patient details and export it to the doctor/patient | have easy access                                                                 |
 
 *{More to be added}*
```
Comment (dayn-1) [inline docs/DeveloperGuide.md:324]: Maybe can delete this line

**Review by dayn-1 — COMMENTED** [review]
LGTM! Thanks @limrik !

**Pull request merged**